src/control/server/storage/bdev/backend_class.go

```diff
@@ -87,9 +87,9 @@ func writeConfFile(log logging.Logger, buf *bytes.Buffer, req *storage.BdevWrite
 		req.OwnerUID, req.OwnerGID)
 }
 
-// writeJSONConf generates nvme config file for given bdev type to be consumed
+// writeJsonConfig generates nvme config file for given bdev type to be consumed
```

Comment:
I know we're not consistent about this, but I thought the linter got cranky about acronyms not being all-caps.

Reply:
I'm feeling it's more consistent to stick with camel case now given that's what we have in the majority of cases